test/system/test_seq_copy_sys.py

```diff
@@ -54,7 +54,7 @@
     ("Copy:transformer:lstm",
      "--encoder transformer --num-layers 2:1 --rnn-cell-type lstm --rnn-num-hidden 64 --num-embed 32"
      " --rnn-attention-type mhdot --rnn-attention-num-hidden 32 --batch-size 16 --rnn-attention-mhdot-heads 1"
-     " --loss cross-entropy --optimized-metric perplexity --max-updates 5000"
+     " --loss cross-entropy --optimized-metric perplexity --max-updates 6000"
```

Comment:
This test sometimes failed with very close scores.  Looks like it just needed a few more updates.